pkg/framework/framework.go

```diff
@@ -184,19 +198,23 @@ func (f *Framework) BeforeEach(contexts ...string) {
 			gomega.Expect(err).NotTo(gomega.HaveOccurred())
 		}
 
-		// Initializing supported operators on given context
-		// TODO Define new argument to control operators to deploy (or to ignore)
-		ctx.OperatorMap = map[operators.OperatorType]operators.OperatorSetup{}
-		for operatorType, builder := range operators.SupportedOperators {
-			operator, err := builder.NewForConfig(ctx.Namespace, restConfig)
+		// Initializing needed operators on given context
+		ctx.OperatorMap = map[operators.OperatorType]operators.OperatorAccessor{}
+		if f.builders == nil || len(f.builders) == 0 {
+			// populate builders with default values
+			for _, builder := range operators.SupportedOperators {
+				f.builders = append(f.builders, builder)
+			}
+		}
```

Comment:
@mkrutov maybe we can initialize the builders in the "NewFramework" method. But adding a method to customize the builders make sense.

Reply:
Another point to this current approach. When namespaces are being generated by the Framework, Builders passed through SetOperatorBuilders wont be aware of context and namespaces being used right?

I am playing with this branch and trying to come up with something that makes existing tests to pass. Tomorrow we can continue with the discussion.

Reply:
Hum, and now I believe I have an idea for customizing the operator image and version and etc... I will probably do it in a new branch from master.